client/components/ProductDetail/UpdateCart.jsx

```diff
@@ -1,64 +1,86 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import styled from 'styled-components';
-import AddToCart from './AddToCart.jsx';
+import ls from 'local-storage';
```

Comment:
what does this package do? you can access local storage natively, but i didnt read into it much so perhaps theres a justification